packages/material-renderers/src/complex/MaterialAnyOfRenderer.tsx

```diff
@@ -46,12 +50,41 @@ export const MaterialAnyOfRenderer = ({
   cells,
   uischema,
   uischemas,
-}: StatePropsOfCombinator) => {
+  id,
+  data,
+}: CombinatorRendererProps) => {
   const [selectedAnyOf, setSelectedAnyOf] = useState(indexOfFittingSchema || 0);
-  const handleChange = useCallback(
-    (_ev: any, value: number) => setSelectedAnyOf(value),
-    [setSelectedAnyOf]
+  const [open, setOpen] = useState(false);
```

Comment:
Minor: This could be renamed to `confirmDialogOpen` or someting similar to immediately make clear what this is about :)